### PR TITLE
Allow to list all nodes again (remove environment filter)

### PR DESCRIPTION
**Pull request by raphink**
Implies and includes https://github.com/puppet-community/puppetboard/pull/184

### CODE REVIEW

**Comment by mcanevet** [comment]
+1

**Comment by nibalizer** [comment]
:+1:

File: puppetboard/app.py [hunk 1]
```diff
@@ -362,16 +384,22 @@ def reports(env, page):
     """
     check_env(env)
 
+    if env == '*':
+        reports_query = None
+        total_query = '["extract", [["function", "count"]]]'
```
Comment (corey-hammerton) [inline puppetboard/app.py]: Only PuppetDB 3.2 supports the count function WITHOUT a query expression. My first implementation of this was like this:

``'["extract", [["function", "count"]], ["~", "certname", ""]]'``

Comment (raphink) [inline puppetboard/app.py]: OK, and this one is backward compatible I suppose

Comment (corey-hammerton) [inline puppetboard/app.py]: This patch that I suggested will be compatible with PuppetDB 3.0 and 3.1. No need to break it.

**Pull request merged**